### ISSUE TITLE
feature request: consolidate bar size and position controls in one screen

### ISSUE THREAD

**Issue by teh-fixerer** [brittyazel/Neuron#172]
bar size and position settings are accessed in different menus.  can this be put on one so you can set them together?

**Comment by brittyazel** [comment]
Yep this is in the GUI rework. I got it ~40% of the way written then got pulled onto other things. It's coming, I just need to find the time to finish it

**Comment by ylixir** [comment]
there are currently no position controls in the gui. i'm going to repurpose this task to add them.

maybe we could also solve #479 by just deleting the microadjust feature if we have the ability to tweak the raw values here
